test/ClangImporter/non-modular-include.swift

```diff
@@ -1,9 +1,8 @@
 // RUN: %empty-directory(%t)
-// RUN: not %target-swift-frontend -typecheck %s -I %S/Inputs/non-modular -F %S/Inputs/non-modular 2>&1 | %FileCheck --check-prefix=CHECK --check-prefix=CHECK-%target-runtime %s
+// RUN: not %target-swift-frontend -enable-objc-interop -typecheck %s -I %S/Inputs/non-modular -F %S/Inputs/non-modular 2>&1 | %FileCheck --check-prefix=CHECK %s
```

Comment:
Please don't change this test; we want to check both error messages. Or add a test with `-disable-objc-interop` instead.

Reply:
Sure, Ill add a second test case :-)